library/src/scala/NamedTuple.scala

```diff
@@ -28,100 +28,26 @@ object NamedTuple:
   extension [V <: Tuple](x: V)
     inline def withNames[N <: Tuple]: NamedTuple[N, V] = x
 
-  export NamedTupleDecomposition.{Names, DropNames}
+  export NamedTupleDecomposition.{
+    Names, DropNames,
+    apply, size, init, last, tail, take, drop, splitAt, ++, map, reverse, zip, toList, toArray, toIArray
+  }
 
   extension [N <: Tuple, V <: Tuple](x: NamedTuple[N, V])
 
     /** The underlying tuple without the names */
     inline def toTuple: V = x
```

Comment:
If we merge this, we should probably add a big warning comment in the code that anything that depends on `toTuple` must be moved to `NamedTupleDecomposition`

Reply:
added the comment